patches/third_party-polymer-v3_0-BUILD.gn.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/third_party/polymer/v3_0/BUILD.gn b/third_party/polymer/v3_0/BUILD.gn
+index b1f40eeb69edec68fdafafa1cfb6a946ef0f3931..11670aea1c1b3be00f131ea82a4b63f0ad96bb82 100644
+--- a/third_party/polymer/v3_0/BUILD.gn
++++ b/third_party/polymer/v3_0/BUILD.gn
+@@ -48,7 +48,7 @@ generate_grd("build_grdp") {
+     "polymer/polymer_bundled.min.js",
+   ]
+ 
+-  if (is_chromeos_ash) {
++  if (!is_android || is_chromeos_ash) {
```

Comment:
@petemill Is there a better way to do this? (commit: https://github.com/brave/brave-core/pull/13498/commits/109b03ab5e6674329b3620bb0cecfa4c9cfceefe). Stems from https://github.com/brave/brave-core/pull/14067